app/android/build.gradle

```diff
@@ -2,11 +2,13 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "31.0.0"
+        buildToolsVersion = "33.0.0"
         minSdkVersion = 21
-        compileSdkVersion = 33
-        targetSdkVersion = 33
+        compileSdkVersion = 34
+        targetSdkVersion = 34
+
         kotlinVersion = "1.8.0"
+
         // use the same NDK version as the Voice SDK
         ndkVersion = "25.2.9519653"
```

Comment:
I know this isn't part of your pr, but do we use the NDK for anything?

Reply:
I'm not sure to be honest. It's used in the default boilerplate, so I'm assuming that React Native needs it.